edb/server/dbview/dbview.pyx

```diff
@@ -535,7 +535,13 @@ cdef class DatabaseConnectionView:
             raise errors.InternalServerError(
                 'no need to serialize state while in transaction')
 
-        ver = self._db.schema_version
+        ver = hash(
```

Comment:
Will `ver` ever end up in Postgres or in another Python process? `hash()` as you know is randomized

Reply:
Negative. In Postgres, it's in the state temporary table and never used.